kpatch-build/create-diff-object.c

```diff
@@ -2109,6 +2148,9 @@ void kpatch_create_hooks_objname_rela(struct kpatch_elf *kelf, char *objname)
 	}
 }
 
+#ifdef __powerpc__
+void kpatch_create_mcount_sections(struct kpatch_elf *kelf) { }
```

Comment:
Actually `kpatch_create_mcount_sections()` converts it *from* a nop to a branch to fentry.  I'm assuming this function isn't needed because the instruction is already a branch on powerpc.